includes/datavalues/SMW_DV_WikiPage.php

```diff
@@ -672,6 +672,13 @@ protected function getShortCaptionText() {
 			$displayTitle = $this->getText();
 		}
 
+		# Remove (context) as per pipe trick.
+		if ( $this->getOption( 'smwgDVFeatures' ) & SMW_DV_WPV_PIPETRICK != 0) {
+			$tc = '[' . Title::legalChars() . ']';
+			$pt = "/($tc+?)( ?\\($tc+\\))/";
+			$displayTitle = preg_replace($pt, '\\1', $displayTitle);
```

Comment:
Whats is the expected output here (or transformation)? Maybe using full variable names `legalChars` and `pattern` for better readibility.

Reply:
Expected transformation is the completion of the pipe trick. Done.